apps/charterafrica/src/components/Dataset/Dataset.js

```diff
@@ -122,6 +122,24 @@ function Dataset({
                 ))}
               </Grid>
             ) : null}
+            <Button
+              variant="contained"
+              color="primary"
+              size="medium"
+              startIcon={<ExternalLinkIcon />}
+              href={url}
+              target="_blank"
+              sx={{
+                width: {
+                  xs: "100%",
+                  sm: "200px",
+                },
```

Comment:
Isn't this button just supposed to be `size="large"`? Or does it have different size?